src/server/apollo.context.spec.ts

```diff
@@ -41,24 +40,96 @@ const TOKEN = jwt.sign(JWT_PAYLOAD, '__secret__', { // IRL, it's a parseable tok
   expiresIn: 1, // second
   subject: AUTH0_ID,
 });
-const CACHE_KEY = `server-core/identity/${ TOKEN }`;
 const USER_ID = 'USER_ID';
 const LOCALE = 'LOCALE';
 const REMOTE_ADDRESS = 'REMOTE_ADDRESS';
 const ME_FRAGMENT = Object.freeze({
   id: 'ME_ID',
 });
-const CACHE = (<unknown>Object.freeze({}) as Cache);
 
-const OTHER_TOKEN = 'OTHER_TOKEN';
-const OTHER_CACHE_KEY = `server-core/identity/${ OTHER_TOKEN }`;
-const SESSION_ID = 'fa44276cb66d302601f14a14ccde5b8ad21994fd92ec0d7b';
+const USER_DATA = {
```

Comment:
```suggestion
const USER_DATA: ContextUserFragment = {
```

that's the shape it should have, right?